docs/api/list_all_user_last_names.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: page
+---
+
+# List all user last names
+
+This page goes over how to list all user last names that are registered on the to-do service.
+
+## Base Endpoint: 
+
+GET /users/last_names
+
+This endpoint retrieves the last names of all users.
+
+### Properties:
+
+  * ' last_names ': Is an array that contains the last names of all users. Specificially an array of strings representing the last names of users.
+
+### Operations: 
+
+  * ' GET /users/last_names ': Retrieves the last names of all users.
+
+#### Examples: 
+
+Request:
+
+curl -X GET https://api.example.com/users/last_names
```

Comment:
I tried this and got:
```
$ curl -X GET https://api.example.com/users/last_names
curl: (6) Could not resolve host: api.example.com
```

I then tried the local URL and got:
```
$ curl -X GET http://localhost:3000/users/last_names
Not Found
```
Are you sure this is the correct syntax and that this operation is supported?